### ISSUE TITLE
RVL-1389 - App closes when adding a structure to operationArea_other

### ISSUE THREAD

**Issue by samkanga** [OpenSRP/opensrp-client-reveal#1197]
Zambia prod
Reveal version 5.3.10 and 5.3.12
Affected users:  dckm1 and more on [this](https://smartregister.atlassian.net/browse/RVL-1389) link.

1. Login to mobile client
2. Select plan and operation area ( or directly select operationArea_other i.e Mayukwayukwa 2 RHC_other )
3. Add structure outside spray areas so that it falls in operationarea_other. i.e Mayukwayukwa 2 RHC_other

**Current Behavior:**
App crashes and closes when I save the added point.  See videos on the link [here](https://smartregister.atlassian.net/browse/RVL-1389).

**Expected behavior:**
The point should be saved under operationarea_other.

**Comment by samkanga** [comment]
@githengi I tested this. I did not encounter a crash. However, when I select save when adding a structure to the `Other` location, the app does not complete the save beyond selecting `Okay` on the pop up that ask if one wants to add the structure in the other location. It goes as far as the screenshot below and does not complete the save action.
![image](https://user-images.githubusercontent.com/5908630/102096681-06e8ac80-3e36-11eb-8988-75a95685086b.png)
